play/src/front/Chat/Connection/Matrix/MatrixChatConnection.ts

```diff
@@ -112,6 +122,8 @@ export class MatrixChatConnection implements ChatConnectionInterface {
         this.client.on(ClientEvent.Room, this.onClientEventRoom.bind(this));
         this.client.on(ClientEvent.DeleteRoom, this.onClientEventDeleteRoom.bind(this));
         this.client.on(RoomEvent.MyMembership, this.onRoomEventMembership.bind(this));
+        this.client.on("RoomState.events" as EmittedEvents, this.onRoomStateEvent.bind(this));
+        this.client.on(RoomEvent.Name, this.onRoomNameEvent.bind(this));
```

Comment:
Try to check that the events registered here are indeed freed when the GameScene is left.

For instance: put a console log in ClientEvent.DeleteRoom event handler. Move from one room to the other a few times, then delete the room. If the console log appears as many times as you switched rooms, we have an issue.